### ISSUE TITLE
feed link does not need "hreflang"

### ISSUE THREAD

**Issue by ivywe** [Geeklog-Core/geeklog#824]
It is error.
feed link does not need "hreflang".

lib-common.php

from:
```
$feed_url[] = '<link rel="alternate" type="' . $format_type
                    . '" hreflang="' . $A['language'] . '" href="'
                    . $baseUrl . $A['filename'] . '" title="'
                    . htmlspecialchars($feed_title) . '"' . XHTML . '>';
```
to:
```
$feed_url[] = '<link rel="alternate" type="' . $format_type
                    .  '" href="'
                    . $baseUrl . $A['filename'] . '" title="'
                    . htmlspecialchars($feed_title) . '"' . XHTML . '>';
```

**Comment by mystralkk** [comment]
Is the current
```PHP
$feed_url[] = '<link rel="alternate" type="' . $format_type
                    . '" hreflang="' . $A['language'] . '" href="'
                    . $baseUrl . $A['filename'] . '" title="'
                    . htmlspecialchars($feed_title) . '"' . XHTML . '>';
```
really causing an error or a warning?

**Comment by ivywe** [comment]
https://support.google.com/webmasters/answer/189077?hl=ja

Multi language site needs hreflang at each site.
I want theme variable without languagecode.
{id_without_lang_id}

then,

top page header

<link rel="alternate" href="{site_url}/index.php?topic=home_ja" hreflang="ja" />
<link rel="alternate" href="{site_url}/index.php?topic=home_en" hreflang="en" />
<link rel="alternate" href="{site_url}/" hreflang="x-default" />

topic top page header

<link rel="alternate" href="{site_url}/index.php?topic={id_without_lang_id}_ja" hreflang="ja" />
<link rel="alternate" href="{site_url}/index.php?topic={id_without_lang_id}_en" hreflang="en" />

 article header

<link rel="alternate" href="{site_url}/article.php/{id_without_lang_id}_ja" hreflang="ja" />
<link rel="alternate" href="{site_url}/article.php/{id_without_lang_id}_en" hreflang="en" />


or

{url_without_lang_id}
e.g.
 http://*****/article.php/xxx

**Comment by mystralkk** [comment]
Fixed with changeset d3ee665

**Comment by shahmirul112** [comment]
d3ee665
